core/modules/installer/installer.manager.inc

```diff
@@ -1141,9 +1139,9 @@ function installer_manager_batch_project_get($project, $url, &$context) {
 /**
  * Determines if file transfers will be performed locally.
  *
- * If the server is configured such that webserver-created files have the same
- * owner as the configuration directory where new code will eventually be
- * installed, the update manager can transfer files entirely locally, without
+ * If the server is configured such that files can written to the configuration
+ * directory where new code will eventually be installed, and the files have
+ * the right ownership, the update manager can transfer files entirely locally, without
  * changing their ownership (in other words, without prompting the user for FTP,
  * SSH or other credentials).
```

Comment:
Could you wrap the next couple of lines to make this fit in 80 characters, per the code style suggestion?